spec/acceptance/user_counters_controller_spec.rb

```diff
@@ -13,7 +13,7 @@
 
   let(:structure) { FactoryBot.attributes_for(:user_counter)[:structure] }
 
-  patch "user/counters/:id" do
+  patch "user/me/counters/:id" do
```

Comment:
Is there an easy way to run this test on both the legacy url and the new url?

Reply:
The options I can think of with my limited ruby/rails knowledge are:
- Duplicate the test code so there is a version for both endpoints
- loop over 2 paths (legacy & current) setting the path to a val and use it in the patch line

Reply:
I was just reading advice yesterday: test code need not be DRY, it should just be obvious. GA and copy/paste ;-)